django/http/request.py

```diff
@@ -99,11 +99,12 @@ def get_host(self):
                 msg += " The domain name provided is not valid according to RFC 1034/1035."
             raise DisallowedHost(msg)
 
-    def get_full_path(self):
+    def get_full_path(self, force_append_slash=False):
```

Comment:
No string opinion on docs for the parameter. Doesn't seem like it would be commonly used in user code.